recipes/dunovo/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+make
+mkdir $PREFIX/include/dunovo
+cp *.py *.so $PREFIX/include/dunovo
```

Comment:
So the Du Novo code expects the `.so` files to be in the same directory as the `.py` files that load them via `ctypes`. I wasn't sure where to stick this mix of files, so I followed what I saw another package do and put it in a special directory under `include`. I could go monkey with the Du Novo code to get it to check `lib` if it doesn't find them in its own directory.